Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.cs

```diff
@@ -22,6 +24,8 @@ namespace Microsoft.Toolkit.Uwp.UI.Extensions
     /// </summary>
     public static class ListViewExtensions
     {
+        private static Dictionary<IObservableVector<object>, Windows.UI.Xaml.Controls.ListViewBase> _itemsForList = new Dictionary<IObservableVector<object>, Windows.UI.Xaml.Controls.ListViewBase>();
```

Comment:
Why not make `using Windows.UI.Xaml.Controls` and remove useless code?

Reply:
Need to fully qualify because of  the ListViewBase extensions class

Reply:
Ok. Fair.